lib/models.js

```diff
@@ -1187,13 +1433,24 @@ class ChoiceValue extends Value {
     str += ')';
     return str;
   }
+ 
+  toJSON() {
+    return Object.assign(super.toJSON(), {
+      "options": this._options.map(v => {
+        let option = Object.assign({}, v.toJSON());
```

Comment:
Is the `Object.assign` necessary?  That would clone the object if you were worried about modifying a _shared_ object, but isn't `v.toJSON()` going to give you a brand new (not shared) object anyway?

Reply:
Definitely not necessary. I probably had some property I was adding to the "option" dictionary that I have since removed.

Reply:
+1